src/rules/get-fullcontact-profile.js

```diff
@@ -22,36 +28,34 @@ function getFullContactProfile(user, context, callback) {
   // skip if no email
   if (!user.email) return callback(null, user, context);
 
-  // skip if fullcontact metadata is already there
-  if (user.user_metadata && user.user_metadata.fullcontact)
-    return callback(null, user, context);
+  // skip if FullContact metadata has already been added to the user profile
+  if (user.user_metadata && user.user_metadata.fullcontact) return callback(null, user, context);
 
-  request.get(
-    'https://api.fullcontact.com/v2/person.json',
-    {
-      qs: {
-        email: user.email,
-        apiKey: FULLCONTACT_KEY
-      },
-      json: true
+  request.post('https://api.fullcontact.com/v3/person.enrich', {
+    'headers': {
+      'Authorization': 'Bearer ' + FULLCONTACT_KEY,
     },
-    (error, response, body) => {
-      if (error || (response && response.statusCode !== 200)) {
-        slack.alert({
-          channel: '#slack_channel',
-          text: 'Fullcontact API Error',
-          fields: {
-            error: error
-              ? error.toString()
-              : response
-              ? response.statusCode + ' ' + body
-              : ''
-          }
-        });
+    body: JSON.stringify({
+      "email": user.email
+    }),
+  }, (error, response, body) => {
+    if (error || (response && response.statusCode !== 200)) {
 
-        // swallow fullcontact api errors and just continue login
-        return callback(null, user, context);
-      }
+      slack.alert({
+        channel: '#slack_channel',
+        text: 'FullContact API Error',
+        fields: {
+          error: error ? error.toString() : (response ? response.statusCode + ' ' + body : '')
+        }
+      });
+
```

Comment:
A `console.log()` would help here. If an email is not found, you get a 404 and no profile.